.azure-pipelines/pipeline.yml

```diff
@@ -0,0 +1,29 @@
+name: $(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+  branches:
+    include:
+      - vscode
+pr: none
+
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: microsoft/vscode-engineering
+      ref: main
+      endpoint: Monaco
+
+parameters:
+  - name: publishPackage
+    displayName: 🚀 Publish @vscode/sudo-prompt
+    type: boolean
+    default: false
+
+extends:
+  template: azure-pipelines/npm-package/pipeline.yml@templates
+  parameters:
+    npmPackages:
+      - name: @vscode/sudo-prompt
```

Comment:
Also please update the name to `sudo-prompt`. Thanks!

Reply:
Done